assets/samplesheet_valid.csv

```diff
@@ -1,2 +1,2 @@
 sample,fastq_1,fastq_2,strandedness
-GM12878,https://github.com/nf-core/test-datasets/raw/modules/data/genomics/homo_sapiens/illumina/fastq/test.rnaseq_1.fastq.gz,https://github.com/nf-core/test-datasets/raw/modules/data/genomics/homo_sapiens/illumina/fastq/test.rnaseq_2.fastq.gz,reverse
+test,https://github.com/nf-core/test-datasets/raw/rnafusion/testdata/human/reads_1.fq.gz,https://github.com/nf-core/test-datasets/raw/rnafusion/testdata/human/reads_2.fq.gz,forward
```

Comment:
Samplesheets and testdata should ideally be hosted on the nf-core/test-datasets on branch rnafusion. The test profiles should then refer to the same url as defined here.

Reply:
[8005728](https://github.com/nf-core/rnafusion/pull/229/commits/80057289d07728c968fa22a0ae6daa23a3d02da5) should address this